zabbix_auto_config/utils.py

```diff
@@ -87,7 +87,9 @@ def validate_host(host):
             assert isinstance(interface["port"], str), "'port' is not a string"
 
     if "inventory" in host:
-        pass  # TODO: What should inventory look like?
+        assert isinstance(host["inventory"], dict), "'inventory' is not a dictionary"
+        for inventory in host["inventory"].items():
+            assert all(isinstance(item, str) for item in inventory), f"Found nonstring value in inventory: {inventory}"
```

Comment:
This isn't wrong, but I think is a bit hard to read due to the "for inventory in inventory". Also: Using `all` on a list of two (key, value) seems a bit unnecessary.

I think I maybe would have done something like:
```python
for key, value in host["inventory"].items():
    assert isinstance(key, str) and isinstance(value, str), f"Found nonstring value in inventory: {key!r}=>{value!r}"
```

Reply:
both er fine and I don't have a strong opinion about it and can use the proposed code.